driver/normalizer/normalizer.go

```diff
@@ -13,7 +15,16 @@ var Preprocess = Transformers([][]Transformer{
 }...)
 
 var Normalize = Transformers([][]Transformer{
-
+	{Mappings(
+		// Move the Leading/TrailingTrivia outside of nodes.
+		//
+		// This cannot be inside Normalizers because it should precede any
```

Comment:
Eventually the SDK should support an easy to use API to move nodes around. Some drivers need to do it for different reasons and currently it's less than ideal (either you do it in the native driver or with custom transforms).

Reply:
Agree. I'm trying different approaches to this in the background, but it still looks like the prototype I did back then might still be a way to go (graphs + copy-on-write).

The biggest problem is how to detect when other transforms should be invoked again, considering the changes made by the current one.